src/main/java/me/leoko/advancedban/manager/CommandManager.java

```diff
@@ -1,5 +1,6 @@
 package me.leoko.advancedban.manager;
 
+import com.imaginarycode.minecraft.redisbungee.RedisBungee;
```

Comment:
I tested I without redis and there is no error, the code of redis is only called when is detected if no, is ignored, if is called and not detected it gonna give a NoClassDefFoundError.
The only error that I have testing without plugins is an error with bStats
`java.lang.IllegalAccessError: tried to access class me.leoko.advancedban.bungeecord.Metrics$1 from class me.leoko.advancedban.bungee.Metrics
    at me.leoko.advancedban.bungee.Metrics.startSubmitting(Metrics.java:167)
    at me.leoko.advancedban.bungee.Metrics.<init>(Metrics.java:102)
    at me.leoko.advancedban.bungee.BungeeMethods.setupMetrics(BungeeMethods.java:131)
    at me.leoko.advancedban.Universal.setup(Universal.java:48)
    at me.leoko.advancedban.bungee.BungeeMain.onEnable(BungeeMain.java:22)
    at net.md_5.bungee.api.plugin.PluginManager.enablePlugins(PluginManager.java:226)
    at net.md_5.bungee.BungeeCord.start(BungeeCord.java:282)
    at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:55)
    at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15)
`
Seems that other plugin was providing it before.